src/language/providers/intellisense/attributeHoverItems.ts

```diff
@@ -0,0 +1,339 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License Version 2.0
+ * (the "License"; you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// prettier-ignore
+
+export function attributeHoverValues(attributeName: string): string {
+  let hoverValue: string = ''
+  switch (attributeName) {
+    case 'name':
+      hoverValue = 'specify name'

```

Comment:
Do we need all the `hoverValue = *` and `break`'s, can you return the string value instead?

Ex:

```
  case 'name':
    return 'specify name'
```

I think that would make this look a bit cleaner, and be less lines, as then each `case` statement only does a `return` instead of doing an assignment plus a `break`. So then you could remove `let hoverValue: string = ''` from the first line of the method and remove `return hoverValue` from the last line of the method.